parachain-tracer/src/tracker.rs

```diff
@@ -408,6 +420,17 @@ impl SubxtTracker {
 			.storage_read_prefixed(CollectorPrefixType::OnDemandOrder(self.para_id), block_hash)
 			.await
 			.map(|v| v.into_inner::<OnDemandOrder>().unwrap());
+
+		if self.on_demand_order.is_none() {
```

Comment:
I'd prefer more canonical `self.on_demand_order.map`.

Reply:
What do you mean in that case?